index.html

```diff
@@ -1,12 +1,339 @@
+                        src="Assets/cart-vector.png"
+                        alt="card"
+                        class="img-fluid"
+                      />
+                    </span>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col-12 col-md-6 col-lg-3 mt-5 mt-lg-0">
+            <div class="card border-0 card-bg-3">
+              <div class="card-body">
+                <h5 class="card-title text-white text-center mt-5 pt-5">
+                  Italian Pasta
+                </h5>
+                <p class="card-text text-white text-center">12,456 &#11088;</p>
+                <p class="text-uppercase text-white">
+                  <small>Mashed potatoes with T-bone steak...</small>
+                </p>
+                <div class="row py-2">
+                  <div class="col-6">
+                    <span class="h5 fw-bold text-center text-white"
+                      >₦ 9,000</span
+                    >
+                  </div>
+                  <div class="col-6 text-end">
+                    <span class="p-2 bg-light rounded-3">
+                      <img
+                        src="Assets/cart-vector.png"
+                        alt="Food cart"
+                        class="img-fluid"
+                      />
+                    </span>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col-12 col-md-6 col-lg-3 mt-5 mt-lg-0">
+            <div class="card border-0 card-bg-4">
+              <div class="card-body">
+                <h5 class="card-title text-white text-center mt-5 pt-5">
+                  Italian Pasta
+                </h5>
+                <p class="card-text text-white text-center">12,456 &#11088;</p>
+                <p class="text-uppercase text-white">
+                  <small>Mashed potatoes with T-bone steak...</small>
+                </p>
+                <div class="row py-2">
+                  <div class="col-6">
+                    <span class="h5 fw-bold text-white">₦ 9,000</span>
+                  </div>
+                  <div class="col-6 text-end">
+                    <span class="p-2 bg-light rounded-3">
+                      <img
+                        src="./images/cart-vector.png"
+                        alt="Food cart"
+                        class="img-fluid"
+                      />
+                    </span>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section id="dishes" class="py-5">
+      <div class="dishes-wrapper">
+        <div class="container1">
+          <div class="row">
+            <div class="col-12">
+              <div class="text-center centered-title mb-5 pb-lg-4">
+                <h2 class="text-black">Some Dishes</h2>
+                <div class="line"></div>
```

Comment:
- [OPTIONAL] This line is aligned to the right in your project, although it should be centered to match the original design
Your design 👇 
![Screenshot from 2022-07-28 12-05-07](https://user-images.githubusercontent.com/87965534/181467234-809f040a-ac5c-4486-9682-d6b58552ca0e.png)
Original design 👇 
![Screenshot from 2022-07-28 12-05-59](https://user-images.githubusercontent.com/87965534/181467324-a15c4b60-091a-47a3-8160-c25c907b4825.png)
Kindly add a `margin: auto` to `line` class, to align it to the center

Reply:
@Sahar-AbdelSamad Thank you for this review. I have fixed it as required.